x-pack/plugins/apm/public/components/app/ErrorGroupOverview/index.tsx

```diff
@@ -103,19 +100,26 @@ const ErrorGroupOverview: React.FC = () => {
         </EuiFlexItem>
         <EuiFlexItem grow={7}>
           <EuiFlexGroup>
```

Comment:
```suggestion
          <EuiFlexGroup gutterSize="s">
```

Will reduce the space between the panels to the same as on the Transactions overview page.

<img width="1453" alt="Screenshot 2020-05-28 at 15 35 19" src="https://user-images.githubusercontent.com/4104278/83148125-d9861c80-a0f8-11ea-8052-e016aab51834.png">